specification/communication/data-plane/SipRouting/preview/2022-09-01-preview/communicationservicessiprouting.json

```diff
@@ -0,0 +1,454 @@
+        },
+        "trunks": {
+          "description": "Gets or sets list of SIP trunks for routing calls. Trunks are represented as FQDN.",
+          "maxLength": 250,
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "SipConfigurationExpanded": {
+      "description": "Represents a SIP configuration.\r\nWhen a call is being routed the routes are applied in the same order as in the routes list.\r\nA route is matched by its number pattern.\r\nCall is then directed into route's first available trunk, based on the order in the route's trunks list. The configuration can be expanded with additional data.",
+      "type": "object",
+      "properties": {
+        "trunks": {
+          "description": "SIP trunks for routing calls.\r\nMap key is trunk's FQDN (1-249 characters).",
+          "maxLength": 250,
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/TrunkExpanded"
+          }
+        },
+        "routes": {
+          "description": "Trunk routes for routing calls.",
+          "maxLength": 250,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TrunkRoute"
+          }
+        }
+      }
+    },
+    "SipConfiguration": {
+      "description": "Represents a SIP configuration.\r\nWhen a call is being routed the routes are applied in the same order as in the routes list.\r\nA route is matched by its number pattern.\r\nCall is then directed into route's first available trunk, based on the order in the route's trunks list.",
+      "type": "object",
+      "properties": {
+        "trunks": {
+          "description": "SIP trunks for routing calls.\r\nMap key is trunk's FQDN (1-249 characters).",
+          "maxLength": 250,
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/Trunk"
+          }
+        },
+        "routes": {
+          "description": "Trunk routes for routing calls.",
+          "maxLength": 250,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TrunkRoute"
+          }
+        }
+      }
+    },
+    "SipConfigurationPatch": {
+      "description": "Represents a SIP configuration patch.",
+      "type": "object",
+      "properties": {
+        "trunks": {
+          "description": "SIP trunks for routing calls.\r\nMap key is trunk's FQDN (1-249 characters).",
+          "maxLength": 250,
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/TrunkPatch"
+          }
+        },
+        "routes": {
+          "description": "Trunk routes for routing calls.",
+          "maxLength": 250,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TrunkRoute"
+          }
+        }
+      }
+    }
+  },
+  "parameters": {
+    "ExpandSipConfiguration": {
+      "name": "expand",
```

Comment:
Maybe you could consider naming this `$expand` instead of `expand`?
That would be in tact with OData standard https://docs.microsoft.com/en-us/aspnet/web-api/overview/odata-support-in-aspnet-web-api/using-select-expand-and-value
Also it would be congruent with other usages of `$expand` in this repository https://github.com/Azure/azure-rest-api-specs/search?q=%22%5C%22name%5C%22%3A+%5C%22%24expand%5C%22%22&type=code

Reply:
Yes those should have never been there